### DISCUSSION TITLE
brush supports ?

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#342] [General]
i know photoshop is supported and duality will be supported for PS imports, soon, too !! 
is there any other brushes you are planning to implement soon ? i know procreate is out of the options, but what about clip studio ? (unless that was already discussed then oops)

**Comment by keiwando** [comment]
Don't have any specific plans for other brush formats at the moment. The vast majority of requests are unsurprisingly for Procreate brushes.

**Reply by trinhpxl** [reply]
understandable xD
